app/src/main/java/ch/blinkenlights/android/vanillaplug/VPlug.java

```diff
@@ -18,43 +18,42 @@
 
 package ch.blinkenlights.android.vanillaplug;
 
-import android.content.SharedPreferences;
 import android.content.Intent;
+import android.content.SharedPreferences;
+import android.content.SharedPreferences.OnSharedPreferenceChangeListener;
 import android.os.Bundle;
 import android.preference.PreferenceActivity;
 import android.preference.PreferenceManager;
 
-import android.util.Log;
-
-public class VPlug extends PreferenceActivity 
-	implements SharedPreferences.OnSharedPreferenceChangeListener {
-
-	@Override
-	public void onCreate(Bundle savedInstanceState) {
-		super.onCreate(savedInstanceState);
-		addPreferencesFromResource(R.xml.preferences);
-		PreferenceManager.getDefaultSharedPreferences(this).registerOnSharedPreferenceChangeListener(this);
-		// Ensure that the service gets started if it was not running
-		tickleService();
-	}
-
-	@Override
-	protected void onDestroy() {
-		super.onDestroy();
-	}
-
-	@Override
-	public void onSharedPreferenceChanged (SharedPreferences sharedPreferences, String key) {
-		tickleService();
-	}
-
-	/**
-	 * Re-Launches the vplug service.
-	 * The service will stay alive (or stop itself) depending on the settings.
-	 */
-	private void tickleService() {
-		Intent intent= new Intent(getApplicationContext(), VPlugService.class);
-		startService(intent);
-	}
+public class VPlug extends PreferenceActivity
```

Comment:
So all this does is change tab to whitespace? what do we gain from this?

Reply:
Nothing, really, except it's more commonplace to use spaces I think.

Reply:
Not in the existing codebase: Both vanilla and the headphone detector use tabs - no need to change this.

Reply:
Hmm ok.

Also while we're on the topic of the main Vanilla app, I feel you should really start grouping the files. All of these are just mixed in with each other, and it can be quite confusing. I was gonna do that myself, but I wasn't sure how you'd want them grouped. https://github.com/vanilla-music/vanilla/tree/master/app/src/main/java/ch/blinkenlights/android/vanilla